README.md

```diff
@@ -1,27 +1,23 @@
-# Datalogics PDF Java Toolkit Sample repository
+# Datalogics PDF Java Toolkit Sample Repository
 
 This repository contains samples for use with [Datalogics PDF Java Toolkit](http://www.datalogics.com/products/pdf/pdfjavatoolkit/).
 
-Please note that even though the samples are MIT licensed, you still require
-a license from Datalogics for PDF Java Toolkit in order to run these samples. Please
-sign up for an evaluation [here](http://www.datalogics.com/products/pdf/pdfjavatoolkit/eval/)
-or [contact us](http://www.datalogics.com/company/contact-us/) to learn more before
-evaluating Datalogics PDF Java Toolkit.
+Please note that even though the samples are MIT licensed, you still require a license from Datalogics for PDF Java Toolkit in order to run them. Please sign up for an evaluation [here](http://www.datalogics.com/products/pdf/pdfjavatoolkit/eval/) or [contact us](http://www.datalogics.com/company/contact-us/) to learn more before evaluating the product.
```

Comment:
Instead of saying _the product_ it should say __Datalogics PDF Java Toolkit__. We want to be specific in what we are asking potential customers to evaluate.